### ISSUE TITLE
Function keys (F1-F12) aren't working as shortcuts properly

### ISSUE THREAD

**Issue by renovatorruler** [binaryage/totalterminal#17]
After the latest update today(8th Aug, 11), my previously configured shortcut key F4 stopped working, when you press the F4 key, the terminal window will show up, but when you press it again to make it disappear, the terminal window eats the shortcut.

None of the Function keys can be used as a shortcut now, they were working fine until the last update.

**Comment by darwin** [comment]
F1-F12 work just fine here on my machine (Lion).

http://totalterminal.binaryage.com/#changelog

The latest version is 1.1.1 (can be seen in the bottom on TotalTerminal Preferences pane). I don't know from which version you've upgraded but in 1.0.1 I've changed the library for setting keyboard shortcuts and updated some other code. 

Please try to rename your Visor profile and create it from scratch. I have a feeling that this F4 eating is caused by some escape codes defined inside your Terminal profile.

**Comment by renovatorruler** [comment]
I upgraded from 1.0.1 to 1.1.1, I created a new visor profile as you recommended but it didn't help me. I tried downgrading totalterminal to 1.0.1 but that didn't help either.

I also deleted the F4 key binding from the visor profile, but that doesn't help either, its not displaying anything on the screen(unlike the other Function keys they display some '~' character), but its eating up the F4 key and not passing it on to TotalTerminal.
